### PR TITLE
feat: User should be able to edit account name when adding an account

### PR DESCRIPTION
**Pull request by juan-cortes**

### CODE REVIEW

File: src/components/SelectableAccountsList.js [hunk 1]
```diff
@@ -112,17 +225,34 @@ class SelectableAccount extends PureComponent<{
         <AccountCard account={account} />
         {!isDisabled && (
           <CheckBox
+            onChange={onPress?() =>onPress(account):undefined}
```
Comment (gre) [inline src/components/SelectableAccountsList.js]: `onPress ? this.onPress : ..`

File: src/components/SelectableAccountsList.js [hunk 2]
```diff
@@ -88,20 +114,107 @@ class SelectableAccountsList extends Component<ListProps> {
...
+    const { rowIndex, listIndex } = this.props;
+    this.panResponder = PanResponder.create({
+      // Ask to be the responder:
+      onStartShouldSetPanResponder: () => true,
+      onStartShouldSetPanResponderCapture: () => false,
+      onMoveShouldSetPanResponder: () => false,
+      onMoveShouldSetPanResponderCapture: () => false,
+
+      onPanResponderGrant: () => {
+        if (swipedAccountSubject) {
```
Comment (gre) [inline src/components/SelectableAccountsList.js]: Why the if?

File: src/components/SelectableAccountsList.js (hunk 1 above)
Comment (juan-cortes) [reply src/components/SelectableAccountsList.js]: how would that pass the account though?

File: src/components/SelectableAccountsList.js (hunk 2 above)
Comment (juan-cortes) [reply src/components/SelectableAccountsList.js]: I was getting undefined if it triggered while we are still scanning, probably something like the component is there but the ref is not ready yet... I don't know. I needed it.

Comment (gre) [reply src/components/SelectableAccountsList.js]: Isn't that variable a global immutable subject? So it's always defined

Comment (juan-cortes) [reply src/components/SelectableAccountsList.js]: don't know what to tell you, if I remove the check it sometimes crashes

**Pull request merged**